cmake/modules/FindLibSync.cmake

```diff
@@ -0,0 +1,37 @@
+# (c) 2014 Copyright ownCloud, Inc.
+# Redistribution and use is allowed according to the terms of the BSD license.
+# For details see the accompanying COPYING* file.
+
+# - Try to find QtKeychain
+# Once done this will define
+#  OWNCLOUDSYNC_FOUND - System has QtKeychain
+#  OWNCLOUDSYNC_INCLUDE_DIRS - The QtKeychain include directories
+#  OWNCLOUDSYNC_LIBRARIES - The libraries needed to use QtKeychain
+#  OWNCLOUDSYNC_DEFINITIONS - Compiler switches required for using LibXml2
```

Comment:
please do not name the variables starting OWNCLOUDSYNC_* but LIBSYNC_*

We need to be brandable where we do not wanna see the name ownCloud anywhere. Not so hot here, admitted, but still ;-)